lib/google.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+create_image_from_snapshot(){
+    IMAGE=$1
+    FAMILY=$2
+    DESCRIPTION=$3
+    SNAPSHOT=$4
+    gcloud compute images create "$IMAGE" --family "$FAMILY" --description "$DESCRIPTION" --source-snapshot "$SNAPSHOT"
+}
+
+create_image_from_bucket(){
+    IMAGE=$1
+    FAMILY=$2
+    DESCRIPTION=$3
+    IMAGE_FILE=$4
```

Comment:
Keep in mind that this sets global variables

Reply:
Yes, let's use local for all the variables that are not meant to be global.